distribution/docs/src/main/resources/content/_architectures/_securityFramework/sts-ldap-login.adoc

```diff
@@ -2,18 +2,18 @@
 :status: published
 :title: Security JAAS LDAP
```

Comment:
❗ I pulled down this branch to generate the docs, and I can't find this section in documentation.html. Same for `Security LDAP Claims Handler, `Claims Handlers`, and `Realms`.

Reply:
Hm... weird ok I'll take another look, thanks.